src/extension.ts

```diff
@@ -55,6 +57,12 @@ async function init(context: vscode.ExtensionContext, git: ApiImpl, gitAPI: GitA
 	const prManager = new PullRequestManager(repository, telemetry, git, credentialStore);
 	context.subscriptions.push(prManager);
 
+	liveshareApiPromise.then((api) => {
```

Comment:
prefer `await` syntax over .then

Reply:
i intentionally changed the 'await' for the 'then', the reason behind is that awaiting the liveshare api means that the liveshare api return would have to wait for the liveshare extension to fully activate. Waiting for the activation would block (awaited fo course) the pr github extension without any good benefit. If you really think that 'await' would be enough i can change the code.

Reply:
Got it, yeah let's not block